matching-game/package.json

```diff
@@ -6,7 +6,8 @@
     "dev": "next dev",
     "build": "next build",
     "start": "next start",
-    "lint": "next lint"
+    "lint": "next lint",
+    "test": "jest ./tests/*"
```

Comment:
this adds a new command, similar to `npm run dev` which starts your script. in this case, you can do `npm run test` and it will run all of the tests.